src/main/java/com/kodilla/ecommercee/OrderController.java

```diff
@@ -1,4 +1,47 @@
 package com.kodilla.ecommercee;
 
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
```

Comment:
Nie używaj  nigdy importu z *. Zawsze wprost podawaj dokładną nazwę klasy lub pola (w import static), którego przestrzeń nazw chcesz zbliżyć do tej klasy. 

Na ten temat możesz poczytać sobie na przykład tutaj:
https://stackoverflow.com/questions/147454/why-is-using-a-wild-card-with-a-java-import-statement-bad

Obecnie każde IDE samo zarządza importami w taki sposób, że nie musisz się tym zbytnio przejmować. 
Jeśli dalej jest niejasne czemu tak nie powinno się robić - napisz, postaram się ten temat rozwinąć.

Reply:
Poprawiłem